cli/internal/util/env/env.go

```diff
@@ -0,0 +1,64 @@
+package env
```

Comment:
I like them because the tests can be right next to the source file, and things are a bit better organized. But defering to you here for the convention.